db/users.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
```

Comment:
Don't think this is needed, but @michael-beaver might be able to shed more light on that.

Reply:
Yeah apparently this isn't necessary anymore with ECMAScript 6